halo2_proofs/src/dev/failure/emitter.rs

```diff
@@ -120,15 +120,18 @@ pub(super) fn expression_to_string<F: Field>(
         &|query| {
             layout
                 .get(&query.rotation.0)
-                .unwrap()
-                .get(
-                    &(
-                        Any::Advice(Advice { phase: query.phase }),
-                        query.column_index,
+                .map(|map| {
+                    map.get(
+                        &(
+                            Any::Advice(Advice { phase: query.phase }),
+                            query.column_index,
+                        )
+                            .into(),
                     )
-                        .into(),
-                )
-                .unwrap()
+                })
+                .flatten()
```

Comment:
I'm curious in what situation we'd find no advice query in the `layout`? Also I couldn't see how this related to the `lookup_columns` fix, could you elaborate more about this change?

Reply:
This error was happening when we have for example an empty MPT table that we load during synthesize time IIRC.
It shouldn't affect anyway the correct  behavior of the error-reporting module. 

It could also happen I think if we use the CellManager and assign values to a Region but never have a `Query` that catches them.

Reply:
I see I see, thanks for explaining!